layouts/partials/tags.html

```diff
@@ -0,0 +1,9 @@
+<div class="tag-group" style="display:inline-block;">
+ {{ range sort .Params.tags }}
+  <div class="tag" style="display:inline-block; padding:5px;">
+    <span class="tag-textlink">
```

Comment:
I'm not seeing a definition for `tag-textlink` -- is this an artifact from your prior effort?

One thing to note (and this doesn't necessarily need to be addressed in this PR) is that you've changed the html from an unordered list (`ul`) to a `span`. This definitely works, but you could also explore using an ordered list (`ol`) instead of span, and making it horizontal using css, similar to [this example](https://www.w3schools.com/html/tryit.asp?filename=tryhtml_lists_horizontal).

I mention using an ordered list because technically it is ordered (alphabetized) now, but it would also work with an unordered list.

Thoughts?